src/static/css/infusion-docs.css

```diff
@@ -351,24 +347,81 @@ div.infusion-docs-note {
     min-width: 13rem;
 }
 
+@media only screen and (max-width: 50rem) {
```

Comment:
How was 50rem in this media query decided upon? If trying to match Foundation's values, see the values in this document: http://foundation.zurb.com/docs/media-queries.html

Also it may be useful to add a comment so that we know what this query is for?